src/widgets/time-series/histogram-view.js

```diff
@@ -131,6 +136,27 @@ module.exports = cdb.core.View.extend({
     this._chartView.forceResize();
   },
 
+  _clearTooltip: function () {
+    this.$('.js-tooltip').stop().hide();
+  },
+
+  _onValueHover: function (info) {
```

Comment:
Q: Don't we have already a tooltip view for managing this?

Reply:
As @xavijam said, we already have a tooltip view for widgets. But it's not ready for this because it depends on a target. Maybe it's worth that we check what it needs to be used here too, and clean the code repetition we have right now.